service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ReviewCMOControllerAboutToSubmitTest.java

```diff
@@ -99,12 +109,68 @@ void shouldSealPDFAndAddToSealedCMOsListWhenJudgeApprovesOrder() throws Exceptio
             .status(APPROVED)
             .build();
 
+        assertThat(State.ISSUE_RESOLUTION).isNotEqualTo(responseData.getState());
         assertThat(responseData.getDraftUploadedCMOs()).isEmpty();
         assertThat(responseData.getSealedCMOs())
             .extracting(Element::getValue)
             .containsExactly(expectedSealedCmo);
     }
 
+    @Test
```

Comment:
I don't mind massively those tests, but I am not sure if it's the best place for them - I men it seems you test different logic, couldn't this be tested on the service level? Maybe the controller is doing too much (I will check later).